### PR TITLE
Add kpod image and kpod rmi for the handling of container images.

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: docs/kpod-images.1.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+% kpod(8) # kpod-images - Simple management tool for pods and images
+% Dan Walsh
+% March 2016
```
Comment (sameo) [inline docs/kpod-images.1.md]: March 2017

File: docs/kpod-images.1.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+**kpod [GLOBAL OPTIONS] images [OPTIONS]**
+
+# GLOBAL OPTIONS
+
+**--help, -h**
+  Print usage statement
+
+**--display**
+  Only container IDs
```
Comment (sameo) [inline docs/kpod-images.1.md]: Did you mean `Only image IDs` ?

Comment (rhatdan) [inline docs/kpod-images.1.md]: Fixed in code and man page

File: docs/kpod-images.1.md (hunk 1 above)
Comment (rhatdan) [inline docs/kpod-images.1.md]: Fixed here and in kpod-rmi.1.md

File: cmd/kpod/rmi.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+		Description: rmiDescription,
+		Action:      rmiCmd,
+		ArgsUsage:   "IMAGE-NAME-OR-ID [...]",
+	}
+)
+
+func rmiCmd(c *cli.Context) error {
+	args := c.Args()
+	if len(args) == 0 {
+		return fmt.Errorf("container ID must be specified")
```
Comment (TomSweeneyRedHat) [inline cmd/kpod/rmi.go]: Should this be container ID or image ID?  If tweaked here, I think buildah will need a similar tweak.

File: cmd/kpod/images.go [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+		if truncate {
+			fmt.Printf("%-12s %s\n", "IMAGE ID", "IMAGE NAME")
+		} else {
+			fmt.Printf("%-64s %s\n", "IMAGE ID", "IMAGE NAME")
+		}
+	}
+	for _, image := range images {
+		if quiet {
+			fmt.Printf("%s\n", image.ID)
+		} else {
```
Comment (sameo) [inline cmd/kpod/images.go]: nitpick: To have a clearer (imho) code, you may want to do:
```Go
for _, image := range images {
   if quiet {
     fmt.Printf("%s\n", image.ID)
     continue
   }

   names := []string{""}
   [...]
}


```

File: cmd/kpod/rmi.go [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+	store, err := getStore(c)
+	if err != nil {
+		return err
+	}
+
+	for _, id := range args {
+		_, err := store.DeleteImage(id, true)
+		if err != nil {
+			return fmt.Errorf("error removing image %q: %v", id, err)
```
Comment (sameo) [inline cmd/kpod/rmi.go]: Is this the intended behavior ? We return after the first failure when trying to delete several images ?

**Comment by sameo** [comment]
Some nitpicks, otherwise LGTM

File: cmd/kpod/rmi.go (hunk 5 above)
Comment (mrunalp) [inline cmd/kpod/rmi.go]: I think we should collect all the errors and report failures together.

**Comment by sameo** [comment]
LGTM

**Comment by mrunalp** [comment]
Needs rebase.

**Comment by mrunalp** [comment]
LGTM

**Comment by mrunalp** [comment]
@rhatdan We merged the wrong PR here. I think you pushed the shutdown code to this one instead of kpod rmi.

**Pull request merged**